Added dv-pod installation flow

Padraic-O-Mhuiris opened the pull request.

Padraic-O-Mhuiris committed:
Added dv-pod installation flow

bussyjd commented on obolup.sh:
```diff
@@ -1222,6 +1222,96 @@ configure_path() {
+		--set charon.dkgSidecar.image.tag=90a1656 \
+		--set charon.nickname="$nickname" \
+		--set validatorClient.type=prysm \
+		--set validatorClient.config.prysm.acceptTermsOfUse=true \
```
This flag doesn't exist anymore

bussyjd commented on obolup.sh:
```diff
@@ -1222,6 +1222,96 @@ configure_path() {
+		--set charon.dkgSidecar.apiEndpoint=https://obol-api-nonprod-dev.dev.obol.tech \
+		--set network=hoodi \
+		--set charon.operatorAddress="$operator_address" \
+		--set charon.dkgSidecar.image.tag=90a1656 \
```
better pin a :latest here while we are in active dev IMO

Commit:
fix: address review comments for dv-pod installation

- Use :latest tag for dkgSidecar image instead of pinned commit hash
- Remove deprecated prysm.acceptTermsOfUse flag (no longer exists)

Commit:
Merge branch 'main' into dv-pod-installation
